cmdstanpy/model.py

```diff
@@ -452,11 +455,11 @@ def sample(
 
         :param cores: Number of processes to run in parallel. Must be an
             integer between 1 and the number of CPUs in the system.
-            If none then set automatically to `chains` but no more
-            than `total_cpu_count - 2`
+            If none then set automatically to chains but no more
+            than total_cpu_count - 2
```

Comment:
Did we use os.cpu_count / multiprocessing.cpu_count?

Reply:
multiprocessing.cpu_count -

```
multiprocessing.cpu_count()
Return the number of CPUs in the system.
```